### ISSUE TITLE
getWAR and nteams = 27.5

### ISSUE THREAD

**Issue by gjm112** [beanumber/openWAR#78]
In the examples for getWAR, there is an example with nteams=27.5.   How should we explain this?

**Comment by beanumber** [comment]
The issue is that the other WAR systems set the replacement level so that the total WAR in the league sums to roughly 1000 wins. Our construction of replacement level is fundamentally different. 

However, through trial-and-error, if you set the number of teams to 27.5, then you designate fewer replacement level players, and the mean replacement performance improves (?), and thus the total `openWAR` sums to roughly 1000 wins. 

In short, this is a total hack that is only there to match up with a total hack that someone else created.
